orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/instance/AbstractInstancesCheckTask.groovy

```diff
@@ -59,6 +62,16 @@ abstract class AbstractInstancesCheckTask extends AbstractCloudProviderAwareTask
 
   abstract protected boolean hasSucceeded(Stage stage, Map serverGroup, List<Map> instances, Collection<String> interestingHealthProviderNames)
 
+  // When waiting for up instances during a "Deploy" stage, it is OK for the server group to not exist coming into this
+  // task. Instead of failing on a missing server group, we retry the stage until it either succeeds, fails, or times out.
+  // For the other uses of this task, we will fail if the server group doesn't exist.
+  boolean waitForUpServerGroup() {
+    return false
+  }
+
+  long serverGroupWaitTime = TimeUnit.MINUTES.toMillis(10)
```

Comment:
I'd maybe move these up to the top below the other backoff/timeout related properties.